pkg/qemu/qemu.go

```diff
@@ -206,16 +206,12 @@ func Cmdline(cfg Config) (string, []string, error) {
 		}
 		return "", nil, errors.New(errStr)
 	}
+
+	cpu := *y.CPUType
+	args = appendArgsIfNoConflict(args, "-cpu", cpu)
 	switch *y.Arch {
 	case limayaml.X8664:
-		cpu := "qemu64"
-		if isNativeArch(*y.Arch) {
-			cpu = "host"
-		}
-		args = appendArgsIfNoConflict(args, "-cpu", cpu)
-		if isNativeArch(*y.Arch) {
-			args = appendArgsIfNoConflict(args, "-machine", "q35,accel="+accel)
-		} else {
+		if strings.HasPrefix(cpu, "qemu64") {
```

Comment:
Why not just compare `cpu != "host"`

Reply:
Yeah, seems prudent. I don't know either if these work for all cpu types (and if the reasoning to use them applies).